fix_appdata.patch

```diff
@@ -0,0 +1,67 @@
+From 4b839dc4c27ae5783aa53187ed27d648f491de0a Mon Sep 17 00:00:00 2001
+From: Sabri Ünal <yakushabb@gmail.com>
+Date: Thu, 2 May 2024 00:29:54 +0300
+Subject: [PATCH] appdata: Update appdata
+
+- Fix duplicated metadata_license
+- Fix screenshot information
+- Fix updatecontact tag
+- Add developer information via developer tag
+- Add missing launchable tag
+- Add several URLs
+---
+ src/org.gnome.OfficeRunner.appdata.xml.in | 22 +++++++++++++++-------
+ 1 file changed, 15 insertions(+), 7 deletions(-)
+
+diff --git a/src/org.gnome.OfficeRunner.appdata.xml.in b/src/org.gnome.OfficeRunner.appdata.xml.in
+index 835f054..796f86f 100644
+--- a/src/org.gnome.OfficeRunner.appdata.xml.in
++++ b/src/org.gnome.OfficeRunner.appdata.xml.in
+@@ -2,13 +2,12 @@
+ <!-- Copyright 2014 William Moreno Reyes <williamjmorenor@gmail.com>
+      Copyright 2014 Bastien Nocera <hadess@hadess.net> -->
+ 
+-<component type="desktop">
+-  <id type="desktop">org.gnome.OfficeRunner.desktop</id>
++<component type="desktop-application">
++  <id>org.gnome.OfficeRunner</id>
```

Comment:
```suggestion
+  <id>org.gnome.OfficeRunner.desktop</id>
```

Please don't change the id like we discussed https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#renaming-id-tag